### PR TITLE
add some sonoff  devices function

### PR DESCRIPTION
**Pull request by liangjia2019**

### CODE REVIEW

File: src/devices/sonoff.ts [hunk 1]
```diff
@@ -359,6 +390,32 @@ const definitions: Definition[] = [
...
+                description: ' ',
+                valueMin:15,
+                valueMax:65535,
+            }),
+            enumLookup({
+                name: 'motion_sensitivity',
+                lookup: {'low': 1, 'medium': 2, 'high': 3},
+                cluster: 0x0406,
+                attribute: {id: 0x0022, type: 0x20},
+                description: ' ',
```
Comment (Koenkk) [inline src/devices/sonoff.ts:407]: Please fill in a description here

File: src/devices/sonoff.ts [hunk 2]
```diff
@@ -359,6 +390,32 @@ const definitions: Definition[] = [
         toZigbee: [],
         ota: ota.zigbeeOTA,
         exposes: [e.occupancy()],
+        extend: [
+            numeric({
+                name: 'ult_timeout',
+                cluster: 0x0406,
+                attribute: {id: 0x0020, type: 0x21},
+                description: ' ',
```
Comment (Koenkk) [inline src/devices/sonoff.ts:398]: Please fill in a description here

File: src/devices/sonoff.ts [hunk 3]
```diff
@@ -359,6 +390,32 @@ const definitions: Definition[] = [
         toZigbee: [],
         ota: ota.zigbeeOTA,
         exposes: [e.occupancy()],
+        extend: [
+            numeric({
+                name: 'ult_timeout',
```
Comment (Koenkk) [inline src/devices/sonoff.ts:395]: What does `ult` mean?

File: src/devices/sonoff.ts [hunk 4]
```diff
@@ -327,6 +327,18 @@ const definitions: Definition[] = [
         fromZigbee: [fz.ias_contact_alarm_1, fz.battery],
         toZigbee: [],
         ota: ota.zigbeeOTA,
+        extend: [
+            binary({
+                name: 'tamper_private',
```
Comment (Koenkk) [inline src/devices/sonoff.ts:332]: Why use `tamper_private` here and not `tamper`?

File: src/devices/sonoff.ts [hunk 5]
```diff
@@ -327,6 +327,18 @@ const definitions: Definition[] = [
...
+                cluster: 0xFC11,
+                attribute: {id: 0x2000, type: 0x20},
+                description: ' ',
```
Comment (Koenkk) [inline src/devices/sonoff.ts:335]: Please add a description here.

**Comment by liangjia2019** [comment]
Hi Koenkk:
I have changed the description here

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**